### PR TITLE
Fix minor markdown issue in ServiceAccount docs

### PR DESCRIPTION
**Pull request by codedevote**
Fix minor issue in ServiceAccount docs.

### CODE REVIEW

File: content/en/docs/reference/kubernetes-api/authentication-resources/service-account-v1.md [hunk 1]
```diff
@@ -16,7 +16,10 @@ weight: 1
 
 ## ServiceAccount {#ServiceAccount}
 
-ServiceAccount binds together: * a name, understood by users, and perhaps by peripheral systems, for an identity * a principal that can be authenticated and authorized * a set of secrets
+ServiceAccount binds together:
```
Comment (kbhawkey) [inline content/en/docs/reference/kubernetes-api/authentication-resources/service-account-v1.md:19]: Hello @codedevote .
This content is auto-generated by the API reference generator.
/cc @feloy .

**Review by Arhell — COMMENTED** [review]
more info about CLA, check please
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by codedevote** [comment]
> more info about CLA, check please
> https://github.com/kubernetes/community/blob/master/CLA.md

I acutally signed the CLA 6 days ago. Maybe I didn't trigger the bot correctly.

**Comment by codedevote** [comment]
As it seems I may have missed a step, which I now finished. Should be fine now (hopefully).

**Comment by feloy** [comment]
Hi @codedevote , this content is generated from the comments into the Kubernetes sources, changing this file won't have any effect as the changes will be overwritten the next time the files will be generated.

For the moment, there is no way to correct this kind of Markdown issue (as the comments in the sources are not specifically in markdown format).

**Comment by codedevote** [comment]
Hi @feloy, ok, didn't know that. So I guess you can close this one. And just for future reference - is there a guide somewhere that would explain how to fix such minor documentation issues. It seems like I fell into the trap to believe that "Edit this page", (see screensot), which is available on every page in the docs, is actually the way to go... 
image

**Comment by kbhawkey** [comment]
Hi @codedevote .
You bring up a good point.
The layout template from the theme, which includes these links, was recently customized for this site.
The API resources generator could add an attribute to the front matter of each generated page.
When the `auto_generated` attribute is set to true, the "edit this page" links are not displayed.
@feloy, What do you think about adding the `auto_generated` attribute to the generator?

Here is the front matter of a generated `config-api` page that includes the attribute:
```
---
title: WebhookAdmission Configuration (v1)
content_type: tool-reference
package: apiserver.config.k8s.io/v1
auto_generated: true
---
```

**Comment by feloy** [comment]
@kbhawkey  Yes, I'll add this attribute to the generated pages

**Comment by feloy** [comment]
@kbhawkey I've added the `auto_generated` metadata, but I cannot see the expected result, that the **Edit this page** should be removed.

Is this feature implemented or is it a work in progress?

**Comment by kbhawkey** [comment]
@feloy, I thought the related PR was merged (https://github.com/kubernetes/website/pull/27393).
It is now. The generated reference pages that use this attribute should omit the "edit" links.

**Comment by kbhawkey** [comment]
👋 @codedevote .
I am closing this PR.
Thanks for your suggestions!
/close

**Pull request closed without merging**